README.md

```diff
@@ -95,6 +95,15 @@ api.splines.each do |spline|
 end
 ```
 
+Other methods, including `[]` and `fetch` are also available
+
+```ruby
+api.users.each do |user|
```

Comment:
Can we make this example based on splines like the one above? Just a more consistent story.